### DISCUSSION TITLE
ElementsGroup items can't have Tooltips

### DISCUSSION THREAD

**Discussion by cstrat** [mantinedev/mantine#40] [Q&A] [answered]
Not really a *bug* because it is clear in the docs as to which children elements are supported inside `<ElementsGroup>`.

What are your thoughts on how to add `<Tooltip>` to an item in an `<ElementsGroup>`?
I'm also having issues with `<Tooltip>` inside the `rightSection` of an `<Input>` element getting clipped.

Two challenges:

1. If I wrap the `<ActionIcon>` with a tool tip the flex doesn't work
2. Either way the `<Tooltip>` is clipped by the `<Input>` - even with zIndex of 9999

<img width="313" alt="image" src="https://user-images.githubusercontent.com/4209076/117434884-8c4d3480-af70-11eb-846a-ab84c79e99b4.png">

----

Sorry here is a repo with an example:

https://codesandbox.io/s/mantine-tooltip-in-element-s9wkr?file=/src/App.js

Three examples in there.

**Comment by rtivital** [comment, marked as answer]
Nope this is a bug actually, It's not related to ElementsGroup, you can use tooltip with it, it's input which has [overflow:hidden](https://github.com/mantinedev/mantine/blob/master/src/mantine-core/src/Input/Input.styles.ts#L15). This is a good use case and bug should be fixed.

**Reply by cstrat** [reply]
It does break also when using with ElementsGroups - did you see the codesandbox.

I will log a bug and take a look at a PR. This would just be a bug with the JSS for input.

Is there a reason the input had overflow hidden?

**Reply by cstrat** [reply]
Same ElementsGroup - one with Tooltip:

<img width="316" alt="image" src="https://user-images.githubusercontent.com/4209076/117439406-22d02480-af76-11eb-8eb9-7155ad638394.png">

**Reply by rtivital** [reply]
overflow is hidden on input to prevent rightSection overflow (first thing on screenshot), but I think, it will be fine to allow overflow since rightSectionWidth can be set

**Reply by rtivital** [reply]
I've created and issue for this #49 will fix in next patch
